do.go

```diff
@@ -196,6 +202,9 @@ func (d *DO) Where(conds ...Condition) Dao {
 	if err != nil {
 		return d.withError(err)
 	}
+	if len(exprs) == 0 {
```

Comment:
dead code?

Reply:
hhhhha, it's really useful , if drop this code, it will produce sql error:
![image](https://user-images.githubusercontent.com/23723772/146326544-b769be49-a71a-4086-a14b-321b394cd56a.png)